analyzers/tests/SonarAnalyzer.UnitTest/TestCases/OperatorsShouldBeOverloadedConsistently.CSharp11.cs

```diff
@@ -0,0 +1,17 @@
+﻿public class Foo : IOperator<Foo> // Compliant - the '+' operator comes IOperator interface
```

Comment:
The rule only ever complains about missing equals/not equals operator and equals, GetHashcode methods. Put the equals operators in the interface (or use [`IEqualityOperators<Foo, Foo, bool>`](https://learn.microsoft.com/en-us/dotnet/api/system.numerics.iequalityoperators-3?view=net-7.0)).